src/index.html

```diff
@@ -1,53 +1,68 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta name="viewport" content="width=device-width, initial-scale=1">
-    <title>Document Two Sections</title>
-    <link rel="stylesheet" href="style.css">
-  </head>
-  <body>
-    <main>
+
+<head>
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  <title>Document Two Sections</title>
+  <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+  <main>
     <div class="side-bar">
       <ul>
         <li><a href="index.html" id="home-link">home</a></li>
         <li><a href="signin.html" id="sign-in-link">Sign in</a></li>
         <li><a href="signup.html" id="sign-up-link">Sign Up</a></li>
         <li><a href="messages.html" id="message">Messages</a></li>
       </ul>
-      </div>
+    </div>
 
 
     <div class="main-body-container">
       <h1>Excepteur Sint Occaecat</h1>
-    <p id="paragraph1">Lorem ipsum <strong>dolor</strong> sit <em>amet</em>, <b>consectetur</b> adipiscing <i>elit</i>. Duis condimentum sagittis ex</p>
-    <ul class="list">
-      <li class="">
-        <a href="#">first point</a>
-      </li>
-      <li class="">
-        <a href="#">second point</a>
-      </li>
-      <li>
-        <a href="#"> <!-- use .list-item-link class if you are confused about nested selectors -->
-          third point
-        </a>
-      </li>
-    </ul>
-    <h2 class="point">Duis Aute</h2>
-    <p>Quis nostrud exercitation ullamco <a href="https://yahoo.com" target="_blank">laboris</a> nisi ut aliquip ex ea commodo consequat.</p>
-    <img src="https://www.rd.com/wp-content/uploads/2016/01/04-dog-breeds-dalmation.jpg" width="150" alt="a Dalmatian dog" id="thumbnail">
-    <button type="button" onclick="nextImage()">Next Image</button>
-    <button type="button" onclick="bark()">🐶🤚Pat the dog</button>
-    <audio class="js-audio-player" src="https://github.com/nbktechworld/introduction-to-html/raw/abcdac58651ba724c40665c82dd55f171534115f/document-two-sections/barking.mp3"></audio>
-    <ol>
-      <li>first point</li>
-      <li>second point</li>
-      <li>third point</li>
-    </ol>
-    <video src="http://commondatastorage.googleapis.com/gtv-videos-bucket/sample/BigBuckBunny.mp4" controls width="300"></video>
+      <p id="paragraph1">Lorem ipsum <strong>dolor</strong> sit <em>amet</em>, <b>consectetur</b> adipiscing
```

Comment:
Do you mind reverting these changes not related to your issue?

Reply:
Hey, thanks for the review, sure I'll fix it right away